sysl2/sysl/binexprEval.go

```diff
@@ -106,7 +107,8 @@ func makeKey(op sysl.Expr_BinExpr_Op, lhs, rhs valueType) string {
 	return fmt.Sprintf("%s_%s_%s", op, lhs.String(), rhs.String())
 }
 
-func (op DefaultBinExprStrategy) eval(txApp *sysl.Application, assign Scope, binexpr *sysl.Expr_BinExpr) *sysl.Value {
+func (op DefaultBinExprStrategy) eval(txApp *sysl.Application,
+	assign syslutil.Scope, binexpr *sysl.Expr_BinExpr) *sysl.Value {
```

Comment:
Move `) *sysl.Value {` to its own line, adding a `,` to the end of this one.

Reply:
Done